gamebot/Program.cs

```diff
@@ -224,20 +235,22 @@ public void Start()
 				});
 			_client.Ready += (s,e) =>
```

Comment:
Oh. Well, you weren't loading the current player. I'll have a look into ready, but it works on D.NET 1.0.0 :P